src/libs/actions/User.js

```diff
@@ -256,6 +257,22 @@ function deletePaypalMeAddress() {
     Growl.show(Localize.translateLocal('paymentsPage.deletePayPalSuccess'), CONST.GROWL.SUCCESS, 3000);
 }
 
+function triggerNotifications(onyxUpdates) {
+    _.each(onyxUpdates, (update) => {
+        if (!update.shouldNotify) {
+            return;
+        }
+
+        const reportID = update.key.replace(ONYXKEYS.COLLECTION.REPORT_ACTIONS, '');
```

Comment:
Noob question, but what is `replace` doing here?

Reply:
The key on the update will be `reportActions_{reportID}` so we're parsing the report ID by removing that reportActions_ prefix (replacing it with empty string).

Reply:
Ah that makes sense now, Thanks!